lib/foreman/renderer/scope/macros/base.rb

```diff
@@ -334,6 +334,17 @@ def gem_version_compare(first, second)
             Gem::Version.new(first.to_s) <=> Gem::Version.new(second.to_s)
           end
 
+          apipie :method, 'Returns content of Foreman SSL certificate' do
```

Comment:
This is incorrect. The setting is defined as this:
https://github.com/theforeman/foreman/blob/ba3da753f5c68ea487dee6834fb39e362e2e9e1f/app/models/setting/auth.rb#L13

So the `ssl_ca_file` setting only contains the CA file for the Smart Proxies. In a default setup we happen to use the same CA, but if you're using a custom certificate on Foreman itself then you're not guaranteed to be able to connect.

Reply:
Why don‘t we simply add a dedicated setting for the ca cert?